Sources/Extensions/UIKit/UIScrollViewExtensions.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  UIScrollViewExtensions.swift
+//  SwifterSwift
+//
+//  Created by camila oliveira on 22/04/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(UIKit)
+    import UIKit
+#if !os(watchOS)
+
+// MARK: - Methods
+public extension UIScrollView{
+    //Original Source: https://gist.github.com/thestoics/1204051
+    /// SwifterSwift: Takes a snapshot of an entire ScrollView
+    ///
+    ///    AnySubclassOfUIScroolView().snapshot
+    ///    UITableView().snapshot
+    ///
+    /// - Returns: Snapshot as UIimage for rendered ScrollView
+    public var snapshot: UIImage?{
+        UIGraphicsBeginImageContextWithOptions(contentSize, false, 0)
+        defer{
+            UIGraphicsEndImageContext()
+        }
+        guard let context = UIGraphicsGetCurrentContext() else { return nil }
+        let previousFrame = frame
+        frame = CGRect(origin: frame.origin, size: contentSize)
+        layer.render(in: context)
+        frame = previousFrame
+        layer.render(in: context)
```

Comment:
This second render call is really necessary? I think it will render the image with the frame size and not the content :))

Reply:
Hi, 
That was my mistake :) . Fixed it. 
Thank you